llvm/lib/Support/Debug.cpp

```diff
@@ -38,25 +40,52 @@
 
 using namespace llvm;
 
+/// Parse a debug type string into a pair of the debug type and the debug level.
+/// The expected format is "type[:level]", where the level is an optional
+/// integer.
+static std::pair<std::string, int> parseDebugType(StringRef DbgType) {
+  int Level = 0;
+  size_t ColonPos = DbgType.find(':');
+  if (ColonPos != StringRef::npos) {
+    StringRef LevelStr = DbgType.substr(ColonPos + 1);
+    DbgType = DbgType.take_front(ColonPos);
+    if (LevelStr.empty())
+      Level = -1;
```

Comment:
Mm, so 'a' would enable but empty disables? E.g., why not just disable by not listing and treating any error as level 0/1?